eng/common/TestResources/deploy-test-resources.yml

```diff
@@ -47,6 +48,8 @@ steps:
       displayName: Deploy test resources
       env:
         TEMP: $(Agent.TempDirectory)
+        ${{ each var in parameters.EnvVars }}:
```

Comment:
For consistency we should add this to the else clause as well as the remove templates.